README.md

```diff
@@ -88,7 +88,7 @@ Based on latest (pre) release.
 | authentication    | Configure HTTP authentication. `basic` or `digest`. Use this with username and password fields.                           | False    |         | string          |
```

Comment:
I have some startup errors I need to investigate before completing this PR.

Reply:
@danieldotnl, the errors are due to configuration being incorrect, so please update that. I fixed it locally for me.